server/hedley/hedley.info

```diff
@@ -55,6 +55,8 @@ dependencies[] = title
 dependencies[] = token
 dependencies[] = views
 dependencies[] = views_bulk_operations
+; We can't add this module locally because it's causing SimpleTest to stall.
```

Comment:
That's a WTF I'd be happy yo avoid. Worth investing more time, and having it working properly.

Reply:
Happy to, @noamoss Can I have another hour to check what's up with this?